components/webui/tests/misc.js

```diff
@@ -0,0 +1,34 @@
+import assert from "assert";
+
+import {unquoteString} from "/imports/utils/misc";
+
+describe("misc", function () {
+    it("unquoteString", function () {
+        // Unquoted string
```

Comment:
It will be good to ensure 100% test coverage of the function.
```suggestion
        // Empty string
        assert.strictEqual(unquoteString("", "\"", "\\"), "");
        // Unquoted string
```